test/e2e/tests/test_scenarios.py

```diff
@@ -90,6 +239,37 @@ class ClusterHelper(helper.ResourceHelper):
     Overrides methods as required for custom resources.
     """
 
+    def assert_items_boto3(self, expectations: dict, actual: dict) -> None:
+        if not expectations:
+            # nothing to assert as there are no expectations
+            return
+        if not actual:
+            # there are expectations but no given actual state to validate
+            # following assert will fail and assert introspection will provide useful information for debugging
+            assert expectations == actual
+
+        clusterName = ""
+        for (latestKey, latestValue) in actual.items():
+            if latestKey == "name":
+                clusterName = latestValue
+        clusters = mdb.describe_clusters(ClusterName=clusterName)
+        cluster = clusters['Clusters'][0]
```

Comment:
QQ: what if the cluster didn't get created? Is it correct to assume that `describe_clusters` will always return a non empty earray?

Reply:
If cluster didn't get created, it will return exception `MemoryDB.Client.exceptions.ClusterNotFoundFault` at line 255.